test-framework/junit5/src/main/java/io/quarkus/test/junit/QuarkusTestExtension.java

```diff
@@ -130,6 +131,10 @@ private ExtensionState doJavaStart(ExtensionContext context, Class<? extends Qua
                     rootBuilder.add(testResourcesLocation);
                 }
             }
+            if (Files.exists(testClassLocation.getParent().resolve("testFixtures"))) {
+                rootBuilder.add(testClassLocation.getParent().resolve("testFixtures"));
+            }
```

Comment:
@glefloch was your testcase failing w/o this change? I did try the reproducer attached by the user to verify. But I wasn't sure whether your testcase was also failing.

Reply:
Yes it was failing as well